source/chapters/library.rst

```diff
@@ -866,8 +871,42 @@ different location. Select :guilabel:`Use Default Library` to reset.
   Mac partition mounted in Linux, you can load it and use your iTunes tracks and
   playlists as well.
 
-.. seealso:: External libraries can be disabled under
-             :menuselection:`Preferences --> Library`.
+.. _library-serato:
+
+Using the Serato library
+------------------------
+
+Mixxx supports reading your Serato library and crates both from your local hard drive as well as portable USB drives. Smart crates are currently not supported.
+
+Serato stores information such as hot cues, track color and beatgrid in the file tags.
+Regardless of whether a track is loaded into deck from the libray or directly from the file browser, Mixxx will import that information automatically if present.
+
+Saved loops are imported as well, but since Mixxx is lacking support for saved loops, only the first loop is usable in Mixxx.
+All other saved loops can be used like regular hotcues for now.
+   
+Waveforms, Gain values and "Flips" are not imported from Serato.
+
+.. note:: Due to differences in the metadata format, importing Serato information from Ogg Vorbis files is currently not supported.
+
+**How to import Serato libraries from external USB drive**
+
+* Copy music library from Serato into external USB drive
```

Comment:
since this is an ordered list of steps, we should use numbers instead of bullet points.

```
  1.  Copy music...
  1.  Plug in USB drive....
```

markdown will set the numbers correctly, you can make all of them "1." and it will work right.

Reply:
This is RST, not Markdown. Just number them correctly :D

Reply:
Hello @ywwg can you request for a change with the numbered list so i commit them directly here? I lost my system (Stolen) I don't have access to the changes I have made already.

Reply:
done!